### PR TITLE
Task update

### PR DESCRIPTION
**Pull request by joshualeeee**

### CODE REVIEW

File: packages/express-backend/backend.js
Comment (MichaelSaunichev) [inline packages/express-backend/backend.js:1]: good job implementing server functionality for retrieving tasks

File: packages/express-backend/task-services.js
Comment (shravansheth) [inline packages/express-backend/task-services.js:1]: Looks good

**Comment by ghost** [comment]
Looks great! Will make frontend integration easy. All useful endpoints.

File: packages/express-backend/task-services.js
Comment (ghost) [reply packages/express-backend/task-services.js:1]: Thank you!

**Pull request merged**